cbpfc.go

```diff
@@ -470,8 +471,9 @@ type packetGuardOpts struct {
 	// requiredGuard returns:
 	// - the packetGuard needed by insns
 	// - the number of instructions in insns covered by the guard.
-	//   The guard is assumed to be invalidated for the remaining / uncovered insns (eg RegX was clobbered for indirect guards).
+	//   The guard is assumed to be invalidated for the next insns (eg RegX was clobbered for indirect guards).
```

Comment:
Why `invalidated` not `valid`?

Reply:
If the returned number of insns was `!= 0` that meant the guard wasn't valid for the next instructions. Reworked the comments to be clearer hopefully.